queue_services/business-emailer/src/business_emailer/config.py

```diff
@@ -0,0 +1,137 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""All of the configuration for the service is captured here.
+
+All items are loaded, or have Constants defined here that
+are loaded into the Flask configuration.
+All modules and lookups get their configuration from the
+Flask config, rather than reading environment variables directly
+or by accessing this configuration directly.
+"""
+import os
+
+from dotenv import find_dotenv, load_dotenv
+
+# this will load all the envars from a .env file located in the project root (api)
+load_dotenv(find_dotenv())
+
+CONFIGURATION = {
+    "development": "legal_api.config.DevConfig",
+    "testing": "legal_api.config.TestConfig",
+    "production": "legal_api.config.ProdConfig",
+    "default": "legal_api.config.ProdConfig"
+}
+
+
+def get_named_config(config_name: str = "production"):
+    """Return the configuration object based on the name.
+
+    :raise: KeyError: if an unknown configuration is requested
+    """
+    if config_name in ["production", "staging", "default"]:
+        config = ProdConfig()
+    elif config_name == "testing":
+        config = TestConfig()
+    elif config_name == "development":
+        config = DevConfig()
+    else:
+        raise KeyError(f"Unknown configuration: {config_name}")
+    return config
+
+
+class Config:  # pylint: disable=too-few-public-methods
+    """Base class configuration that should set reasonable defaults.
+
+    Used as the base for all the other configurations.
+    """
+
+    # used to identify versioning flag
+    SERVICE_NAME = "emailer"
+    PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+    MSG_RETRY_NUM = int(os.getenv("MSG_RETRY_NUM", "5"))
+
+    SENTRY_DSN = os.getenv("SENTRY_DSN") or ""
+    SENTRY_DSN = "" if SENTRY_DSN.lower() == "null" else SENTRY_DSN
+    LD_SDK_KEY = os.getenv("LD_SDK_KEY", None)
+
+    # urls
+    DASHBOARD_URL = os.getenv("DASHBOARD_URL", None)
+    AUTH_WEB_URL = os.getenv("AUTH_WEB_URL", None)
+    NOTIFY_API_URL = os.getenv("NOTIFY_API_URL", None)
+    LEGAL_API_URL = os.getenv("LEGAL_API_URL", None)
+    PAY_API_URL = os.getenv("PAY_API_URL", None)
+    AUTH_URL = os.getenv("AUTH_URL", None)
+    ACCOUNT_SVC_AUTH_URL = os.getenv("ACCOUNT_SVC_AUTH_URL", None)
+    # secrets
+    ACCOUNT_SVC_CLIENT_ID = os.getenv("ACCOUNT_SVC_CLIENT_ID", None)
+    ACCOUNT_SVC_CLIENT_SECRET = os.getenv("ACCOUNT_SVC_CLIENT_SECRET", None)
+    # variables
+    LEGISLATIVE_TIMEZONE = os.getenv("LEGISLATIVE_TIMEZONE", "America/Vancouver")
+    TEMPLATE_PATH = os.getenv("TEMPLATE_PATH", None)
```

Comment:
This is where you set the default for TEMPLATE_PATH and it should fix all the current test errors in the CI